### DISCUSSION TITLE
Concurrent connections per app?????

### DISCUSSION THREAD

**Discussion by chinamingsss** [ueberdosis/tiptap#5923] [General Questions]
“Concurrent connections per app” displayed on the homepage of the official website
What does it mean? Does it refer to the number of people editing documents online at the same time? If there is no multi-person collaboration function but only personal editing of personal documents but there is a comment function, how to calculate this value?

**Comment by nperez0111** [comment]
concurrent connections refer to number of websocket connections to a document server. So roughly the number of users connected to any document at a given time
